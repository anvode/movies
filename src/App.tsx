import React from 'react';
import { Route, Switch } from 'react-router-dom';

import Movies from './components/Movies';
import MovieDetail from './components/MovieDetail';

import './App.scss';

function App() {
    return (
        <>
            <Switch>
                <Route exact path='/' component={Movies} />
                <Route path='/:id' component={MovieDetail}/>
            </Switch>
        </>
    );
}

export default App;
