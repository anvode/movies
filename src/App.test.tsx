import React from 'react';
import Store from './redux/store';
import { render } from '@testing-library/react';
import App from './App';
import {Provider} from 'react-redux';
import { BrowserRouter } from 'react-router-dom';

describe('App', () => {
    it('snapshot App', () => {
        const { asFragment } = render( <Provider store={Store}><BrowserRouter><App /></BrowserRouter></Provider>);
        expect(asFragment()).toMatchSnapshot();
    });
});