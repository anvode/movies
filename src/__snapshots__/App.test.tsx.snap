// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot App 1`] = `
<DocumentFragment>
  <div
    class="loader__container"
  >
    <div
      class="loader"
    >
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
</DocumentFragment>
`;
